components/tag/index.css

```diff
@@ -12,195 +12,190 @@
+	--spectrum-tag-border-color-hover: var(--highcontrast-tag-border-color-hover, var(--mod-tag-border-color-hover, transparent));
+	--spectrum-tag-border-color-active: var(--highcontrast-tag-border-color-active, var(--mod-tag-border-color-active, transparent));
+	--spectrum-tag-border-color-focus: var(--highcontrast-tag-border-color-focus, var(--mod-tag-border-color-focus, transparent));
+	--spectrum-tag-border-color-emphasized: var(--highcontrast-tag-border-color-emphasized, var(--mod-tag-border-color-emphasized, transparent));
+	--spectrum-tag-border-color-disabled: var(--highcontrast-tag-border-color-disabled, var(--mod-tag-border-color-disabled, transparent));
 
 	/* background color */
-	--spectrum-tag-background-color: var(--spectrum-gray-50);
-	--spectrum-tag-background-color-hover: var(--spectrum-gray-50);
-	--spectrum-tag-background-color-active: var(--spectrum-gray-100);
-	--spectrum-tag-background-color-focus: var(--spectrum-gray-50);
-
-	/* border radius */
-	--spectrum-tag-size-small-corner-radius: var(--spectrum-corner-radius-100);
-	--spectrum-tag-size-medium-corner-radius: var(--spectrum-corner-radius-100);
-	--spectrum-tag-size-large-corner-radius: var(--spectrum-corner-radius-100);
-
-	/* border color */
-	--spectrum-tag-border-color: var(--spectrum-gray-700);
-	--spectrum-tag-border-color-hover: var(--spectrum-gray-800);
-	--spectrum-tag-border-color-active: var(--spectrum-gray-900);
-	--spectrum-tag-border-color-focus: var(--spectrum-gray-800);
+	--spectrum-tag-background-color: var(--highcontrast-tag-background-color, var(--mod-tag-background-color, var(--spectrum-gray-100)));
+	--spectrum-tag-background-color-hover: var(--highcontrast-tag-background-color-hover, var(--mod-tag-background-color-hover, var(--spectrum-gray-200)));
+	--spectrum-tag-background-color-active: var(--highcontrast-tag-background-color-active, var(--mod-tag-background-color-active, var(--spectrum-gray-200)));
+	--spectrum-tag-background-color-focus: var(--highcontrast-tag-background-color-focus, var(--mod-tag-background-color-focus, var(--spectrum-gray-200)));
+
+	--spectrum-tag-background-color-selected: var(--highcontrast-tag-background-color-selected, var(--mod-tag-background-color-selected, var(--spectrum-neutral-background-color-default)));
+	--spectrum-tag-background-color-selected-hover: var(--highcontrast-tag-background-color-selected-hover, var(--mod-tag-background-color-selected-hover, var(--spectrum-neutral-background-color-hover)));
+	--spectrum-tag-background-color-selected-active: var(--highcontrast-tag-background-color-selected-active, var(--mod-tag-background-color-selected-active, var(--spectrum-neutral-background-color-down)));
+	--spectrum-tag-background-color-selected-focus: var(--highcontrast-tag-background-color-selected-focus, var(--mod-tag-background-color-selected-focus, var(--spectrum-neutral-background-color-key-focus)));
```

Comment:
This doesn't match the token spec because the token used there doesn't appear to exist.